autovalet_gazebo/launch/aruco_sim.launch

```diff
@@ -0,0 +1,32 @@
+<!-- To run simulation correctly along with this launch file:
+Download the package aruco_gazebo for the markers from here: https://github.com/joselusl/aruco_gazebo.git
+We import aruco_visual_marker_7 model in our world
+Detection done using aruco_ros package from pal-robotics: https://github.com/pal-robotics/aruco_ros.git 
+Put the packages in ~catkin_ws/src/-->
+
+
+<launch>
+
+    <arg name="markerId"        default="582"/>
+    <arg name="markerSize"      default="0.034"/>    <!-- in m -->
+    <arg name="eye"             default="left"/>
+    <arg name="marker_frame"    default="aruco_marker_frame"/>
+    <arg name="ref_frame"       default=""/>  <!-- leave empty and the pose will be published wrt param parent_name -->
+    <arg name="corner_refinement" default="LINES" /> <!-- NONE, HARRIS, LINES, SUBPIX -->
+
+
+    <node pkg="aruco_ros" type="single" name="aruco_single">
+        <remap from="/camera_info" to="/realsense/frontCamera/color/camera_info" />
+        <remap from="/image" to="/realsense/frontCamera/color/image_raw" />
```

Comment:
Now that @SubramanianKrish 's MR has been merged, should those topic names be without `/realsense`?

Reply:
I will rebase